field_friend/automations/implements/implement.py

```diff
@@ -42,3 +50,17 @@ def settings_ui(self):
 
     def reset_kpis(self):
         """Reset KPIs for the implement."""
+    
+    def _update_time(self):
+        """Update KPIs for time"""
+        if not self.is_active:
+            return
+        if self.start_time is None:
+            self.start_time = rosys.time()
+        passed_time = rosys.time() - self.start_time
+        if passed_time > 1:
+            self.system.kpi_provider.increment_all_time_kpi('time')
+            self.start_time = rosys.time()
+            self.start_time = rosys.time()
+            self.start_time = rosys.time()
+            self.start_time = rosys.time()
```

Comment:
```suggestion
            self.start_time = rosys.time()
```
What happened here? 😄

Reply:
I Have no idea 😄

Reply:
i dont know.